specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2023-08-01-preview/SpringEnvironments.json

```diff
@@ -0,0 +1,446 @@
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Collection of resources.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SpringEnvironment"
+          }
+        },
+        "nextLink": {
+          "description": "Link to next page of resources.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "SpringEnvironment": {
+      "description": "An environment for hosting Azure Spring Apps",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "Spring environment resource specific properties",
+          "type": "object",
+          "properties": {
+            "provisioningState": {
+              "description": "Provisioning state of the Service",
+              "enum": [
+                "Creating",
+                "Updating",
+                "Deleting",
+                "Deleted",
+                "Succeeded",
+                "Failed"
+              ],
+              "type": "string",
+              "readOnly": true,
+              "x-ms-enum": {
+                "name": "ProvisioningState",
+                "modelAsString": true
+              }
+            },
+            "serviceId": {
+              "description": "A GUID which uniquely identifies an Azure Spring Apps Environment",
+              "type": "string",
+              "readOnly": true
+            },
+            "managedEnvironmentId": {
+              "description": "The resource Id of the Managed Environment that the Spring Apps instance builds on",
+              "type": "string"
+            },
+            "infraResourceGroup": {
+              "description": "The name of the resource group that contains the infrastructure resources",
+              "type": "string"
+            },
+            "zoneRedundant": {
+              "description": "Whether or not this Spring Environment is zone-redundant.",
+              "type": "boolean",
+              "x-ms-mutability": [
+                "create",
+                "read"
+              ]
+            },
+            "defaultDomain": {
+              "description": "Default Domain Name for the spring environment",
+              "type": "string",
+              "readOnly": true
+            }
+          },
+          "sku": {
+            "description": "Sku of Azure Spring Apps",
+            "type": "object",
+            "properties": {
+              "name": {
+                "description": "Name of the Sku",
+                "type": "string",
+                "default": "S0"
+              },
+              "tier": {
+                "description": "Tier of the Sku",
+                "type": "string",
+                "default": "Standard"
+              }
+            }
```

Comment:
@bowen5 Do we still need SKU? As we are creating a new resource type springEnvironment. 
If we still need it, I guess we can reference the official SKU definition provided by ARM instead of being align with the previous one?

Reply:
Per offline discussion. I will remove the sku.